### ISSUE TITLE
Magento reCaptcha not accessible

### ISSUE THREAD

**Issue by anqaka** [SnowdogApps/magento2-alpaca-theme#115]
Default Magento reCaptcha is not accessible, we should implement google reCaptcha:
https://developers.google.com/recaptcha/
https://github.com/magespecialist/m2-MSP_ReCaptcha
https://github.com/magento/magespecialist_ReCaptcha/

We should add this in theme dependencies and implement on demo page.

**Comment by Aekal** [comment]
From Magento version 2.3 google recaptcha module is installed by default.
Deployed to demo site with badge style fix.
